src/components/ProjectTypeSelection/ProjectTypeSelection.js

```diff
@@ -33,23 +34,20 @@ class ProjectTypeSelection extends PureComponent<Props> {
       <FadeIn>

```

Comment:
I feel like the `FadeIn` and `FormField` shouldn’t be in this component so that it’s the same top-level markup as `ProjectIconSelection`.

Reply:
Yes, you're right. We should be consistent here. I'll change this.